src/popup/base.js

```diff
@@ -38,7 +38,7 @@ export const elements = {
   homeIcon: document.getElementById('home-icon'),
   primarySection: document.querySelector('.section-primary'),
   bookmarksSection: document.querySelector('.section-bookmarks'),
-  deleteAllBookmarksButton: document.querySelector('.delete-all-button'),
+  deleteSelectedBookmarksButton: document.querySelector('.delete-all-button'),
```

Comment:
Since we have changed the context of the "delete" button, change `deleteSelectedBookmarksButton` -> `deleteBookmarksButton`.

Reply:
Hey, @makkoncept I have changed the code as you directed